tools/markdown-to-html/docs-marked-renderer.ts

```diff
@@ -49,15 +49,31 @@ export class DocsMarkdownRenderer extends Renderer {
    * Method that will be called whenever inline HTML is processed by marked. In that case,
    * we can easily transform the example comments into real HTML elements. For example:
    *
-   *  `<!-- example(name) -->` turns into `<div material-docs-example="name"></div>`
+   * `<!-- example(
+   *   {
+   *    "example": "exampleName",
+   *    "file": "example-html.html",
+   *    "lines": "[5, 10]",
```

Comment:
The value shouldn't be quoted here, since we want it to be a `number[]`, not a `string`, same for `true`